script.js

```diff
@@ -0,0 +1,32 @@
+const rock = "rock";
+const paper = "paper";
+const scissors = "scissors";
+const compVal = ['rock','paper','scissors'];
+let humanVal;
+
+function computerPlay() {   
+    let rand = Math.floor(Math.random()*compVal.length);
+    return compVal[rand];  
+}
+
+function humanPlay() {
+    let humanVal = prompt("Enter value:");
+    return humanVal;  
+}
```

Comment:
We don't even need a variable here, so I guess `return prompt("Enter value:");` should be fine

Reply:
fixed